_data/toc.yaml

```diff
@@ -606,10 +608,12 @@ reference:
     title: Get started
   - path: /engine/api/sdks/
     title: SDKs
-  - path: /engine/api/v1.32/
-    title: v1.32 Reference
-  - sectiontitle: Previous API versions
+  - path: /engine/api/latest/
+    title: v{{ latest_stable_docker_engine_api_version }} Reference
+  - sectiontitle: API reference by version
     section:
+    - path: /engine/api/v1.32/
```

Comment:
When we add an API version we only have to put it here. This is no longer "previous" but "all" versioned APIs. The one above is always just "latest", which will also be in this list by number.